src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -9,97 +9,238 @@
+    String webhookSection = "Webhook Overrides";
+
     @ConfigSection(
         name = "Collection Log",
         description = "Settings for notifying about collection log",
-        position = 1
+        position = 0
     )
     String collectionSection = "Collection Log";
 
     @ConfigSection(
         name = "Pet",
         description = "Settings for notifying when obtaining a pet",
-        position = 4
+        position = 10
     )
     String petSection = "Pet";
 
     @ConfigSection(
         name = "Levels",
         description = "Settings for notifying when levelling a skill",
-        position = 7
+        position = 20
     )
     String levelSection = "Levels";
 
     @ConfigSection(
         name = "Loot",
         description = "Settings for notifying when loot is dropped",
-        position = 11
+        position = 30
     )
     String lootSection = "Loot";
 
     @ConfigSection(
         name = "Death",
         description = "Settings for notifying when you die",
-        position = 16
+        position = 40
     )
     String deathSection = "Death";
 
     @ConfigSection(
         name = "Slayer",
         description = "Settings for notifying when you complete a slayer task",
-        position = 21
+        position = 50
     )
     String slayerSection = "Slayer";
 
     @ConfigSection(
         name = "Quests",
         description = "Settings for notifying when you complete a quest",
-        position = 24
+        position = 60
     )
     String questSection = "Quests";
 
     @ConfigSection(
         name = "Clue Scrolls",
         description = "Settings for notifying when you complete a clue scroll",
-        position = 27
+        position = 70
     )
     String clueSection = "Clue Scrolls";
 
     @ConfigSection(
         name = "Speedruns",
         description = "Settings for notifying when you finish a speedrun",
-        position = 32
+        position = 80
     )
     String speedrunSection = "Speedruns";
 
     @ConfigSection(
         name = "Kill Count",
         description = "Settings for notifying when you kill a boss",
-        position = 37
+        position = 90
     )
     String killCountSection = "Kill Count";
 
     @ConfigSection(
         name = "Combat Tasks",
         description = "Settings for notifying when you complete a combat achievement",
-        position = 43
+        position = 100
     )
     String combatTaskSection = "Combat Tasks";
 
     @ConfigSection(
         name = "Achievement Diary",
         description = "Settings for notifying when you complete an Achievement Diary",
-        position = 47
+        position = 110
     )
     String diarySection = "Achievement Diary";
 
     @ConfigItem(
-        keyName = "discordWebhook",
-        name = "Discord Webhook",
-        description = "The Webhook URL for the discord channel",
-        position = 0
+        keyName = "discordWebhook", // do not rename; would break old configs
+        name = "Primary Webhook URL",
```

Comment:
Not super-relevant to this PR, but since we're making a name change could this maybe just be `Primary Webhook URLs` to give users a hint that they can input multiple URLs? Should be added to the description too. "You can use multiple webhooks by putting their URLs on separate lines" or something